src/wgpu.rs

```diff
@@ -299,18 +310,27 @@ impl ProfileContext {
 				label: Some("Tracy Query Resolve"),
 			});
 			for pool in &mut frame.pools {
-				encoder.resolve_query_set(&pool.query, 0..(pool.used_queries as u32), &pool.readback, 0);
+				encoder.resolve_query_set(&pool.query, 0..(pool.used_queries as u32), &pool.resolve, 0);
+				encoder.copy_buffer_to_buffer(&pool.resolve, 0, &pool.readback, 0, pool.resolve.size());
```

Comment:
Could you make this only copy the number of queries used, instead of the whole buffer? (and the same in `with_enabled`)

Reply:
Good point! Added in https://github.com/SparkyPotato/tracy_full/pull/3/commits/5216594e6f5006bcf5e6befd7c7894b35e2d5bfd.